Woof/Woof/ViewModel/LoginViewModel/SelectedRole.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+/// Roles that the user can select on the Login screen
+enum SelectedRole {
```

Comment:
Why do we need this `enum`?
I thought we had an agreement to refactor the architecture later. Please revert it back and simplify for now.

Reply:
it was a wrapping enum in order not to use the enum from the model in view

Reply:
Agreed with the team that we will refactor under mvvm later